src/test/java/bio/terra/service/resourcemanagement/ProfileConnectedTest.java

```diff
@@ -160,8 +160,8 @@ public void testAzureInvalidMissingParams() throws Exception {
 
         var invalidUuidRequest = ProfileFixtures.randomBillingProfileRequest()
             .cloudPlatform(CloudPlatform.AZURE)
-            .tenantId(null)
-            .subscriptionId(null)
+            .tenantId("not-a-valid-uuid")
```

Comment:
Just want to be careful...I think this a good test but I think we also want the test for null

Reply:
This is an outdated diff, but also I'm not sure why this even showed up as an addition because this is what is currently on dev: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/test/java/bio/terra/service/resourcemanagement/ProfileConnectedTest.java#L163